src/actions/exchangeActions.js

```diff
@@ -54,6 +53,7 @@ import { getSmartWalletAddress } from 'utils/accounts';
 import { reportErrorLog, reportLog } from 'utils/common';
 import { getAssetsAsList, getAssetData, isSynthetixTx } from 'utils/assets';
 import { isOrderAmountTooLow } from 'utils/exchange';
+import httpRequest from 'utils/httpRequest';
```

Comment:
Consider changing naming to more standard one.
```suggestion
import httpClient from 'utils/httpClient';
```